src/Pages/EditMember.jsx

```diff
@@ -0,0 +1,138 @@
+import { useState } from "react";
+import Button from "@mui/material/Button";
+import Box from "@mui/material/Box";
+import TextField from "@mui/material/TextField";
+
+export default function EditMember(props) {
+  return (
+    <section className="two-column-grid-forms">
+      <div className="available-classes">
+        <h2>Available Classes: </h2>
+        <ul>
+          {/* {props.map((oneClass, index) => {
+            const {} = props; */}
```

Comment:
Same as above

Reply:
this was part of the file setup
will have to pass data through props at some point
thanks